main.go

```diff
@@ -370,21 +371,27 @@ func graphql(query string, variables map[string]interface{}, responseData interf
 }
 
 type options struct {
-	cwd      string
-	baseRef  string
-	headRef  string
-	format   string
-	filename string
-	author   string
-	print    func(notifs map[string][]string) error
+	cwd                 string
+	baseRef             string
+	headRef             string
+	format              string
+	filename            string
+	subscriberThreshold int
+	author              string
+	print               func(notifs map[string][]string) error
 }
 
 func markdownCommentTitle(filename string) string {
 	return fmt.Sprintf("<!-- codenotify:%s report -->\n", filename)
 }
 
 func (o *options) writeNotifications(w io.Writer, notifs map[string][]string) error {
-	subs := []string{}
+	if o.subscriberThreshold > 0 && len(notifs) > o.subscriberThreshold {
+		fmt.Fprintf(w, "The number of notifying subscribers (%d) has exceeded the threshold (%d).\n", len(notifs), o.subscriberThreshold)
```

Comment:
```suggestion
		fmt.Fprintf(w, "Not notifying subscribers because the number of notifying subscribers (%d) has exceeded the threshold (%d).\n", len(notifs), o.subscriberThreshold)
```